### PR TITLE
[DOCS] Expose URI parts processor in Painless

### PR DESCRIPTION
**Pull request by danhermann**
Documents the new Painless method added in #73634.

Relates to #73346

Preview: https://elasticsearch_73896.docs-preview.app.elstc.co/guide/en/elasticsearch/painless/master/painless-ingest.html

### CODE REVIEW

File: docs/painless/painless-guide/painless-ingest.asciidoc [hunk 1]
```diff
@@ -96,3 +96,14 @@ supplied in the `value` parameter.
 ----
 String urlDecode(String value);
 ----
+
+===== URI decomposition
+Use the {ref}/uriparts-processor.html[URI parts processor] to decompose the URI string
```
Comment (danhermann) [inline docs/painless/painless-guide/painless-ingest.asciidoc:101]: From looking at the existing documentation for the other processors, I don't see how these link references are resolved.

Comment (lockewritesdocs) [reply docs/painless/painless-guide/painless-ingest.asciidoc:101]:
```suggestion
Use the {ref}/uri-parts-processor.html[URI parts processor] to decompose the URI string
```

Comment (lockewritesdocs) [reply docs/painless/painless-guide/painless-ingest.asciidoc:101]: @danhermann, looking at the syntax, it looks like you're missing a dash in the URL: https://www.elastic.co/guide/en/elasticsearch/reference/master/uri-parts-processor.html

**Review by lockewritesdocs — APPROVED** [review]
LGTM 👍 Thanks for adding this @danhermann

**Comment by danhermann** [comment]
Thanks, @lockewritesdocs!

**Pull request merged**